package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eth-proof",
-  "version": "2.1.2",
+  "version": "2.1.3",
```

Comment:
good bot, but you must still bump the version in the `package.json` file in order to finish the job!